Tests/KnitTests/DependencyBuilderTests.swift

```diff
@@ -18,9 +18,9 @@ final class DependencyBuilderTests: XCTestCase {
         XCTAssertEqual(
             builder.dependencyTree.debugDescription,
             """
-            Assembly3
```

Comment:
Moving to a Set loses the input ordering for debug descriptions, it's now alphabetically ordered